graylog2-web-interface/src/components/indices/IndexSetConfigurationForm.jsx

```diff
@@ -14,21 +14,43 @@
  * along with this program. If not, see
  * <http://www.mongodb.com/licensing/server-side-public-license>.
  */
+// @flow strict
 import PropTypes from 'prop-types';
 import React from 'react';
 import moment from 'moment';
 import lodash from 'lodash';
 import { PluginStore } from 'graylog-web-plugin/plugin';
 
+import HideOnCloud from 'util/conditional/HideOnCloud';
 import { LinkContainer } from 'components/graylog/router';
 import { Col, Row, Button } from 'components/graylog';
 import { Input } from 'components/bootstrap';
 import { Spinner, TimeUnitInput } from 'components/common';
 import IndexMaintenanceStrategiesConfiguration from 'components/indices/IndexMaintenanceStrategiesConfiguration';
-import {} from 'components/indices/rotation'; // Load rotation plugin UI plugins from core.
-import {} from 'components/indices/retention'; // Load rotation plugin UI plugins from core.
-
-class IndexSetConfigurationForm extends React.Component {
+// import {} from 'components/indices/rotation'; // Load rotation plugin UI plugins from core.
+// import {} from 'components/indices/retention';
```

Comment:
These lines should not be commented out, they provide pluggable entities.

Reply:
Forget to get it back. I commented it out to test something.